api4/cloud.go

```diff
@@ -83,7 +83,7 @@ func getSubscriptionStats(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	count, err := c.App.Srv().Store.User().Count(model.UserCountOptions{})
+	count, err := c.App.Srv().GetTotalUsersCount()
```

Comment:
You are calling `GetTotalUsersCount` in the `Srv()` and you should be calling this in the `c.App`.

Reply:
My bad. fixed. @jespino